jest.config.ts

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  preset: 'ts-jest',
+  testEnvironment: 'node',
+  roots: ['<rootDir>/src'],
+  modulePaths: ['<rootDir>/src'],
+};
+
+// module.exports = {
+//   preset: 'ts-jest',
+//   testEnvironment: 'node',
+//   roots: ['<rootDir>/src'],
+//   transform: {
+//     '^.+\\.tsx?$': 'ts-jest',
+//   },
+//   moduleDirectories: ['node_modules', 'src'],
+//   globals: {
+//     'ts-jest': {
+//       tsconfig: 'tsconfig.json',
+//     },
+//   },
+// };
```

Comment:
Remove these if you are not using them. We should get a habit of writing clean codes!